UnityProject/Assets/Scripts/Managers/NetworkManagement/SpawnHandler.cs

```diff
@@ -72,18 +79,18 @@ public static void TransferPlayer(NetworkConnection conn, short playerController
 			playerScript.characterSettings = characterSettings;}
 	}
 
-	private static GameObject CreateGhost(GameObject oldBody)
+	private static GameObject CreateMob(JobType jobType, GameObject spawnSpot, GameObject mobPrefab)
 	{
-		GameObject ghostPrefab = CustomNetworkManager.Instance.ghostPrefab;
-		Vector3 spawnPosition = oldBody.GetComponent<ObjectBehaviour>().AssumedLocation();
```

Comment:
I'm not sure about replacement of AssumedLocation with WorldPosition, it'll probably cause issues when someone dies inside a locker

Reply:
![yep](https://user-images.githubusercontent.com/10403536/59674067-03795400-91b2-11e9-9477-f8942957565d.png)
yep, it causes issues